test/Chainweb/Test/Pact/PactInProcApi.hs

```diff
@@ -582,6 +583,62 @@ pact43UpgradeTest bdb mpRefIO pact = do
         $ mkKeySetData "k" [sender00]
     runCut' = runCut testVersion bdb pact (offsetBlockTime second) zeroNoncer noMiner
 
+chainweb215Test :: TestBlockDb -> IO (IORef MemPoolAccess) -> WebPactExecutionService -> IO ()
+chainweb215Test bdb mpRefIO pact = do
+
+  -- run past genesis, upgrades
+  forM_ [(1::Int)..33] $ \_i -> runCut'
+
+  -- execute pre-fork xchain transfer (blocc0)
+  setOneShotMempool mpRefIO blocc0
+  runCut' -- 34
+  pwo34 <- getPWO bdb cid
+  tx34_0 <- txResult "pwo34_0" 0 pwo34
+  tx34_1 <- txResult "pwo34_1" 1 pwo34
+  assertSatisfies "tx34_0 success" (_pactResult $ _crResult tx34_0) isRight
+  assertSatisfies "tx34_1 success" (_pactResult $ _crResult tx34_1) isRight
+
+  putStrLn "HERE"
```

Comment:
Is this supposed to be HERE?

Reply:
Still working out these tests, so yes for the time being :)